stats/v1/STATS_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,555 @@
+                statusCode:
+                  type: integer
+                  x-stoplight:
+                    id: cy1zgcy127m67
+                  format: int32
+                  example: 200
+                  description: "Http response status code of respective reference call.\r\nFor quality monitoring purposes.  "
+                params:
+                  type: array
+                  x-stoplight:
+                    id: n2s8ohu65wreo
+                  description: The request query parameters being used.
+                  minItems: 0
+                  uniqueItems: true
+                  example:
+                    - eventTypes
+                    - equipmentReference
+                  items:
+                    x-stoplight:
+                      id: 7af29u3bwnubb
+                    type: string
+                uniqueAdopterHash:
+                  type: string
+                  description: "SHA-256 of the adopter. \r\n\r\n> *Rules to be defined.*"
+                  x-stoplight:
+                    id: iem9syusrn81j
+                  example: ba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad
+              required:
+                - domain
+                - method
+                - path
+                - version
+        description: ''
+      tags:
+        - RealTime
+      x-stoplight:
+        id: nh2qnk5aizcue
+    parameters: []
+  /v1/stats-summary:
+    post:
+      summary: Stats Summary
+      operationId: post-v1-stats-summary
+      responses:
+        '204':
+          description: No Content
+          headers:
+            API-Version:
+              $ref: '#/components/headers/API-Version'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+          headers:
+            API-Version:
+              $ref: '#/components/headers/API-Version'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+          headers:
+            API-Version:
+              $ref: '#/components/headers/API-Version'
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+          headers:
+            API-Version:
+              $ref: '#/components/headers/API-Version'
+      x-stoplight:
+        id: 923k81lod9m4g
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/StatSummary'
+        description: ''
+      parameters:
+        - $ref: '#/components/parameters/api-key'
+      description: This endpoint allows you to report the daily usage statistics of a all DCSA APIs being used.
```

Comment:
There is something wrong with the wording in the end of this description. Maybe the `a` should be removed between `of` and `all`...